### DISCUSSION TITLE
What may I do if I want to run milvus (indexing and/or searching) on GPUs/GPU-like chips?

### DISCUSSION THREAD

**Discussion by victorsoda** [milvus-io/milvus#23145] [Q&A and General discussion] [answered]
I am at the very beginning of learning to use Milvus. I meet with two questions as follows.

1. I want to run Milvus (Standalone) on my GPUs, i.e., to use the GPUs to do the calculations of indexing and/or similarity search. Are there any docs or suggestions on this topic?

2. My chip is similar to GPU but different. If I want to run Milvus (Standalone) on a GPU-like chip, what may I do fundamentally? May I try to push some codes to support running Milvus on my chip?

Thanks a million for your kind help.  : - )

**Comment by yhmo** [comment, marked as answer]
- Versions from v2.0.x to v2.2.x cannot support GPU.

- The 2.3.0-beta version can support Nvidia GPU: https://github.com/milvus-io/milvus/releases/tag/v2.3.0-beta
It is still in the beta stage, you can have a try. I believe it only supports Nvidia cards and requires NVIDIA Driver installed.

- The old Milvus version v1.x also supports Nvidia GPU: https://milvus.io/docs/v1.1.1/milvus_docker-gpu.md, it requires NVIDIA Pascal or higher cards and NVIDIA Driver 418 or higher.

**Reply by victorsoda** [reply]
Many thanks for your reply : )
